tools/metrics/types.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"path"
+	"time"
+)
+
+const (
+	ghURL = "https://github.com"
+
+	akoAuthor = "mongodb"
+
+	// #nosec G101 false positive detected by gosec linter, this is not a secret
+	ako = "mongodb-atlas-kubernetes"
+
+	akoWorkflowFilename = "test.yml"
+
+	runsPathFmt = "%s/actions/runs/%d"
+)
+
+const (
+	// PerPage is the results per page on paged queries
+	PerPage = 100
+)
+
+const (
+	Weekly = 7 * 24 * time.Hour
+
+	DayFormat = "2006/01/02"
+)
+
+type TestType int
+
+const (
+	Unit TestType = iota
+	Integration
+	E2E
+)
+
+func (tt TestType) String() string {
+	if tt == Unit {
```

Comment:
nit: use switch statement for better readability.

Reply:
Done